File: The Dog Walker/The Dog Walker/ScheduleData.swift
//
//  ScheduleData.swift
//  The Dog Walker
//
//  Created by Scott O'Hara on 6/17/17.
//  Copyright © 2017 Scott O'Hara. All rights reserved.
//

import Foundation

class ScheduleData{
    
    //stored properties
    var date: String
    var time: String
    var duration: String
    var petName: String
    var instructions: String
    var meds: String
    var price: String
    var scheduleKey: String?
    var paidFlag: Bool?
    var checkIn: Bool?
    var checkOut: Bool?
    
    
    //init stored properties
    init(date: String, time: String, duration: String, petName: String, instructions: String, meds: String, price: String) {
        
        self.date = date
        self.time = time
        self.duration = duration
        self.petName = petName
        self.instructions = instructions
        self.meds = meds
        self.price = price
    }
    
    
}

File: TheDogWalker/app/src/main/java/com/ohara/thedogwalker/dataModel/ScheduleData.java
package com.ohara.thedogwalker.dataModel;

import java.io.Serializable;


public class ScheduleData implements Serializable {

    //stored properties
    public String date;
    public String time;
    public String duration;
    public String petName;
    public String specialIns;
    public String meds;
    public String price;
    public String scheduleKey;
    public String paidFlag;
    public String checkIn;
    public String checkOut;
    public String breed;

    //firebase default constructor
    public ScheduleData(){
    }

    //constructor
    public ScheduleData(String date, String time, String duration, String petName, String specialIns, String meds, String price, String breed){

        this.date = date;
        this.time = time;
        this.duration = duration;
        this.petName = petName;
        this.specialIns = specialIns;
        this.meds = meds;
        this.price = price;
        this.breed = breed;
    }

    @Override
    public String toString() {
        return date + "                                                                 " + petName;
    }
}
